api/src/changeRequests/processChangeRequest.spec.ts

```diff
@@ -42,15 +42,25 @@ describe("processChangeRequest", () => {
                 _id: "new-language",
                 type: "language",
                 memberOf: ["group-languages"],
+                // Even though acl's are not valid for "language" documents, we include it here for unit testing purposes.
```

Comment:
As discussed, then this field shouldn't be passed through to the database with the change request. Investigate if this is a class transformer issue, as it should convert to a class instance which presumably only has the fields you declare.

Reply:
Agree. This actually uncovered a bigger issue with DTO validation, thanks!